crates/core/src/subscription/module_subscription_actor.rs

```diff
@@ -170,34 +182,13 @@ impl ModuleSubscriptions {
             }
             EventStatus::OutOfEnergy => {} // ?
         }
-    }
 
-    /// A blocking version of [`broadcast_event`][Self::broadcast_event].
-    pub fn blocking_broadcast_event(
-        &self,
-        client: Option<&ClientConnectionSender>,
-        subscriptions: &SubscriptionManager,
-        event: Arc<ModuleEvent>,
-    ) {
-        tokio::runtime::Handle::current().block_on(self.broadcast_event(client, subscriptions, event))
-    }
-
-    /// Broadcast the commit event to all interested subscribers.
-    ///
-    /// This function is blocking, even though it returns a future. The returned future resolves
-    /// once all updates have been successfully added to the subscribers' send queues (i.e. after
-    /// it resolves, it's guaranteed that if you call `subscriber.send(x)` the client will receive
-    /// x after they receive this subscription update).
-    fn broadcast_commit_event(
-        &self,
-        subscriptions: &SubscriptionManager,
-        event: Arc<ModuleEvent>,
-        sender_client: Option<&ClientConnectionSender>,
-    ) {
-        subscriptions.eval_updates(&self.relational_db, event, sender_client)
+        Ok(Ok(event))
     }
 }
 
+pub struct WriteSkew;
```

Comment:
nit: Maybe better name is `WriteConflict` because by defination `WriteSkew` anomoly is something transaction doesn't really detect while commiting unless transaction is tracking its reads.